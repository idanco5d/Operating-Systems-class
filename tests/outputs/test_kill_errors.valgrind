==2027== Memcheck, a memory error detector
==2027== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==2027== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==2027== Command: /mnt/c/Users/illus/os/hw/1/smash
==2027== Parent PID: 2022
==2027== 
==2027== 
==2027== FILE DESCRIPTORS: 4 open (3 std) at exit.
==2027== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_kill_errors.valgrind
==2027==    <inherited from parent>
==2027== 
==2027== 
==2027== HEAP SUMMARY:
==2027==     in use at exit: 0 bytes in 0 blocks
==2027==   total heap usage: 95 allocs, 95 frees, 85,808 bytes allocated
==2027== 
==2027== All heap blocks were freed -- no leaks are possible
==2027== 
==2027== For lists of detected and suppressed errors, rerun with: -s
==2027== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
