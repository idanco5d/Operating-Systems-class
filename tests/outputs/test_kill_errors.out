smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash: sending SIGKILL signal to 1 jobs:
9433: sleep 4&
